### PR TITLE
Init arcx

### PR DESCRIPTION
**Pull request by tomquirk**
## What does this PR do and why?

Api key set on mainnet vercel (not goerli)

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/hooks/Wallet/hooks/UserAddress.ts [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
   const [{ wallet }] = useConnectWallet()
   const userAddress = useMemo(() => wallet?.accounts[0]?.address, [wallet])
+
+  useEffect(() => {
+    if (!userAddress) return
+
+    getArcxClient().then(arcx => {
+      arcx?.connectWallet({ account: userAddress, chain: readNetwork.chainId })
+    })
+  }, [userAddress])
```
Comment (tomquirk) [inline src/hooks/Wallet/hooks/UserAddress.ts:16]: Not sure if this is the best place to put this.

Comment (wraeth-eth) [reply src/hooks/Wallet/hooks/UserAddress.ts:16]: I think this probably belongs in a different hook, wdyt?

File: src/lib/arcx.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { ArcxAnalyticsSdk } from '@arcxmoney/analytics'
+
+const ARCX_API_KEY = ''
+
+let client: ArcxAnalyticsSdk
+
+export async function getArcxClient() {
```
Comment (wraeth-eth) [inline src/lib/arcx.ts:7]: why not turn this into a hook?

Comment (tomquirk) [reply src/lib/arcx.ts:7]: because im lazy :D . will do, good callout 🙏

**Review by wraeth-eth — APPROVED** [review]

**Pull request merged**